dronesym-frontend/src/app/drone-list/drone-list.component.ts

```diff
@@ -26,7 +26,7 @@ export class DroneListComponent {
 
     this.droneFeed.getDroneFeed()
     .subscribe((drones) => {
-        // console.log(drones);
+       console.log(drones);
```

Comment:
Console logs are not a good idea. Let's remove them when the db_migration is complete and being merged with the develop.